### PR TITLE
Mes 2863 ui tweaks

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
A lot of tiny UI changes

### CODE REVIEW

**Comment by buzzub** [comment]
IMG_0103

IMG_0104

IMG_0105

File: src/theme/sass-partials/_input.scss [hunk 1]
```diff
@@ -36,3 +37,10 @@ ion-select.select {
     color: inherit;
   }
 }
+
+ion-input.input-ios .text-input-clear-icon {
+  right: 16px;
+  background-image: url("data:image/svg+xml;charset=utf-8,<svg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20512%20512'><polygon%20fill='%23222222'%20points='405,136.798%20375.202,107%20256,226.202%20136.798,107%20107,136.798%20226.202,256%20107,375.202%20136.798,405%20256,285.798%20375.202,405%20405,375.202%20285.798,256'/></svg>");
```
Comment (Cavalier07) [inline src/theme/sass-partials/_input.scss]: Does this work offline? - think the image will need to be stored in the app

Comment (buzzub) [reply src/theme/sass-partials/_input.scss]: The image is an SVG, so it's just data :)

Comment (buzzub) [reply src/theme/sass-partials/_input.scss]: Paste it into your (offline) browser window to see `data:image/svg+xml;charset=utf-8,<svg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20512%20512'><polygon%20fill='%23222222'%20points='405,136.798%20375.202,107%20256,226.202%20136.798,107%20107,136.798%20226.202,256%20107,375.202%20136.798,405%20256,285.798%20375.202,405%20405,375.202%20285.798,256'/></svg>`

Comment (ammarhaiderbjss) [reply src/theme/sass-partials/_input.scss]: Would prefer to have it stored in `src/assets/imgs` alongside all the other images we use in the app which would also make it reusable. Then this reference would just be `assets/clear-icon.svg` or something along those lines.

Comment (Cavalier07) [reply src/theme/sass-partials/_input.scss]: @buzzub - Didn't know that, learn new things

The cross isn't a proper cross though , theres something wrong with it when you open it up big

File: src/pages/test-results-search/components/advanced-search/advanced-search.ts [hunk 2]
```diff
@@ -50,6 +50,6 @@ export class AdvancedSearchComponent {
   }
 
   private dateToString(date: any) {
-    return moment().year(date.year).month(date.month).day(date.day).format('YYYY-MM-DD');
+    return moment().year(date.year).month(date.month - 1).date(date.day).format('YYYY-MM-DD');
```
[the remark this answers is not in the document]
Comment (szabolcs-keresztes) [reply src/pages/test-results-search/components/advanced-search/advanced-search.ts]: The purpose of the DateTime helper is to deal with the date time travel variable coming from the config microservice. As that is not involved here we can just easily use `moment`.

File: src/theme/sass-partials/_input.scss (hunk 1 above)
Comment (buzzub) [reply src/theme/sass-partials/_input.scss]: > Would prefer to have it stored in `src/assets/imgs` 
Yeah, makes sense. In this instance I copied the CSS from the MD ionic styling, which had the image inline.

File: src/pages/test-results-search/components/advanced-search/advanced-search.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (johnmorleybjss) [reply src/pages/test-results-search/components/advanced-search/advanced-search.ts]: I agree that we should use a `DateTime` instance here.

**Comment by szabolcs-keresztes** [comment]
Okay, I'll move that code to `DateTime` helper

**Pull request merged**